jupyter-notebooks/orders_basemaps/README.md

```diff
@@ -0,0 +1,26 @@
+# Ordering Basemaps Using SDK v2 Beta and Requests Library
+
+## Preparing your Workspace 
+
+Follow the set up instructions below to run the Jupyter Notebooks for this workshop.
+
+### Option 1: (recommended) Run in Google Colab
+
+Interact directly with this repo's Jupyter Notebooks by running them in Google Colab.
+
+Each Notebook will have its own **"Open in Colab"** button: once running on Colab, you'll need to run a quick setup cell in each notebook (this will install prerequisites and download data into your Colab workspace). You can also choose to make a copy of the Notebook in your own Google Drive, if you want to save any changes you make (not required for this workshop).
+
+### Option 2: Run local Jupyter instance
+
+You can instead choose to open this Notebook in your own local Jupyter instance. In that case, you'll need to clone this repository and install/download prerequisites.
+
+Clone this repo:
+```bash
+git clone git@github.com:planetlabs/notebooks.git
+cd notebooks/jupyter-notebooks/orders_basemaps/
+```
+
+### About this notebook
+
+These notebooks describe two different ways to download basemaps using the Orders API. The first file demonstrates using the SDK v2 Beta. This is the way we recommend ordering Planet Basemaps. Using the SDK v2 Beta has functions ready-made to make working with the Planet Orders API more efficient and less prone to errors. This means less lines of code and less work for you. Check out the SDK v2 Beta docs [here](https://planet-sdk-for-python-v2.readthedocs.io/en/latest/python/sdk-guide/). The second file demonstrates using the Python Requests Library. We recommend using the SDK v2 Beta. It will cut down on time and improve accuracy. 
```

Comment:
In this section, to emphasize that these notebooks are showing 2 different ways of doing the same thing, I'd recommend doing something like "Method 1: SDK (recommended)" with a link there to that notebook and then a description below that, and the same for Method 2. 

Even if someone just skims the readme, this way they'll be able to have a general idea of what's going on with these notebooks